docs/USAGE-GUIDE.md

```diff
@@ -77,35 +77,57 @@ And when invoking CMake for your project, do one of two things:
  1. Set the `CMAKE_INSTALL_PREFIX` variable with the path to your s2n-tls build.
  2. If you have globally installed s2n-tls, do nothing, it will automatically be found.
 
-## Building s2n-tls with OpenSSL-1.1.1
-
-To build s2n-tls with OpenSSL-1.1.1, do the following:
+## Building s2n-tls with Openssl
 
+We keep the build artifacts in the -build directory:
```

Comment:
Nit: What did the "-" here mean?
```suggestion

We keep the build artifacts in the build directory:
```

Reply:
updated to `*-build`

https://github.com/aws/s2n-tls/blob/main/libcrypto-build/README.md